source/backends/debian/debpkgindex.d

```diff
@@ -112,7 +114,9 @@ public:
                                             "Translation-%s.%s".format(lang, "%s"));
 
             try {
-                fname = downloadIfNecessary (rootDir, tmpDir, fullPath);
+                synchronized (this) {
+                    fname = downloadIfNecessary (rootDir, tmpDir, fullPath);
```

Comment:
Meh, hardly; the code basically just returns the filename (after checking if it exists) if it's not remote or if it exists.

If you insist, then I have to look at the filename in `debpkgindex.d` which makes it more ugly (currently remote file handling is all taken care of in utility methods).